app/sim_test.go

```diff
@@ -29,58 +32,70 @@ var (
 	blockSize int
 	enabled   bool
 	verbose   bool
+	commit    bool
 )
 
 func init() {
 	flag.Int64Var(&seed, "SimulationSeed", 42, "Simulation random seed")
-	flag.IntVar(&numBlocks, "SimulationNumBlocks", 500, "Number of blocks")
+	flag.IntVar(&numBlocks, "SimulationNumBlocks", 50, "Number of blocks")
 	flag.IntVar(&blockSize, "SimulationBlockSize", 200, "Operations per block")
-	flag.BoolVar(&enabled, "SimulationEnabled", false, "Enable the simulation")
+	flag.BoolVar(&enabled, "SimulationEnabled", true, "Enable the simulation")
 	flag.BoolVar(&verbose, "SimulationVerbose", false, "Verbose log output")
+	flag.BoolVar(&commit, "SimulationCommit", false, "Have the simulation commit")
```

Comment:
SimulationNumBlocks & SimulationEnabled should not be modified, if SimulationEnabled is true, unit test may deal with sim test

Reply:
ok, i will set SimulationEnabled = false